src/test/java/apimatic/core/utilities/CoreHelperTest.java

```diff
@@ -137,6 +137,24 @@ public void testBase64Encoding() {
         assertEquals(actualEncodedString, expectedEncodedString);
     }
 
+    @Test
+    public void testBase64EncodingWithNullValue() {
```

Comment:
Blocker: Add assertions when the password is null and when both credentials are null.

Reply:
done